specifications/raml/api.raml

```diff
@@ -412,8 +347,10 @@ traits:
           example: "7ouMYWpwJ422jRcDASZB7P,4VqPOruhp5EdPBeR92t6lQ,2takcwOaAZWiXQijPHIx7B"
       responses:
         200:
-          body:
-            text/plain:
+        403:
```

Comment:
I think the way to do that is make it a trait, maybe a data type, and apply it to _all_ the endpoints. But I'll look into it further because that seems like a very verbose way of handling it.